scripts/download_data.py

```diff
@@ -280,14 +293,36 @@ def downloadFastas(self):
                 'display': 'fasta',
                 'range': '{}-{}'.format(0, maxPos)})
             url = '{}%26{}'.format(path, args)
+            tempFileName = '{}.fa.temp'.format(chromosome)
             fileName = '{}.fa'.format(chromosome)
-            downloader = utils.HttpFileDownloader(url, fileName)
+            downloader = utils.HttpFileDownloader(url, tempFileName)
             downloader.download()
+            # We need to replace the header on the downloaded FASTA
+            with open(tempFileName, "r") as inFasta,\
+                    open(fileName, "w") as outFasta:
+                # Write the new header
+                print(">{}".format(chromosome), file=outFasta)
```

Comment:
I can change it if you like, but as you say, it's a script so I wouldn't get too worried about it.